MixItUp.Base/Services/External/StreamlabsService.cs

```diff
@@ -16,35 +16,23 @@ namespace MixItUp.Base.Services.External
 {
     public class StreamlabsDonation
     {
-        [JsonProperty("donation_id")]
-        public int ID { get; set; }
+        [JsonProperty("_id")]
+        public string ID { get; set; }
 
         [JsonProperty("name")]
         public string UserName { get; set; }
+
         [JsonProperty("message")]
         public string Message { get; set; }
 
-        [JsonProperty("amount")]
+        [JsonProperty("formatted_amount")]
```

Comment:
The new websocket provides both formatted and regular amounts. This takes advantage of that.